openhands/runtime/runtime.py

```diff
@@ -42,7 +43,7 @@ def _default_env_vars(sandbox_config: SandboxConfig) -> dict[str, str]:
     return ret
 
 
-class Runtime:
+class Runtime(FileEditRuntimeMixin):
```

Comment:
It's not clear to me why we need to have a MixIn if it's mixed in to the base class? Couldn't we just add it to the base class?

Reply:
The main reason is that i kinda want these editing logic to stay in a separate file until they are finalized so the base class is a little cleaner